test/functional/lua/highlight_spec.lua

```diff
@@ -1,14 +1,187 @@
+local Screen = require('test.functional.ui.screen')
 local helpers = require('test.functional.helpers')(after_each)
-local exec_lua = helpers.exec_lua
+local clear = helpers.clear
+local command = helpers.command
 local eq = helpers.eq
 local eval = helpers.eval
-local command = helpers.command
-local clear = helpers.clear
-
-describe('vim.highlight.on_yank', function()
+local exec_lua = helpers.exec_lua
+local feed = helpers.feed
 
-  before_each(function()
+before_each(function()
     clear()
+end)
+
+describe('highlight.on_yank()', function()
```

Comment:
no reason to change this from fully qualified to a more coy/clever form. fully qualified names are useful for grep etc.
```suggestion
describe('vim.highlight.on_yank()', function()
```